lib/spack/spack/package.py

```diff
@@ -162,6 +162,7 @@ class tty:
 configure: Executable
 make_jobs: int
 make: MakeExecutable
+nmake: Executable
```

Comment:
In `msvc/package.py` you set

```
module.nmake = MakeExecutable("nmake", jobs=1)
```

so I assume this should be a `MakeExecutable`.